nemo/export/vllm/exporter.py

```diff
@@ -0,0 +1,340 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
Can we move this file under export folder like tensorrt_llm.py? And to be consistent with the current class naming? Exporter is a good name but can we change it to vLLM ? That way it'll be consistent with the trt-llm one.

Reply:
Done, the file moved to `../vllm_exporter.py` and the class renamed to `vLLMExporter`